src/entities/viewer/ui/login.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Button } from 'shared';
+import { viewerModel } from 'entities/viewer';
+
+export const LoginButton: React.FunctionComponent = () => {
```

Comment:
just a note: You can use React.FC instead of React.FunctionalComponent

Reply:
Okey, I will.